lib/vanagon/platform/windows.rb

```diff
@@ -107,7 +107,11 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
-        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
+        wix_extensions =  "-ext WiXUtilExtension -ext WixUIExtension"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} #{wix_extensions}"
```

Comment:
Did we decide we want to keep `-dPlatform=#{@architecture}` and use it? Or would it make more sense to always take advantage of erb templates and architecture set there and remove this?